mythril/laser/ethereum/state/world_state.py

```diff
@@ -21,7 +26,7 @@ class WorldState:
     def __init__(
         self,
         transaction_sequence=None,
-        annotations: List[StateAnnotation] = None,
+        annotations: List[WSDependencyAnnotation] = None,
```

Comment:
```suggestion
        annotations: List[StateAnnotation] = None,
```
This shouldn't be changed right?

Reply:
In any case laser should not depend on any of the plugins I think